multicompany_property_account/views/product_views.xml

```diff
@@ -26,11 +26,11 @@
           <group name="accounting_group">
             <field
               name="property_account_income_id"
-              domain="[('deprecated', '=', False),('company_id','=',company_id)]"
+              domain="[('deprecated', '=', False),'|', ('company_id','=',company_id), ('company_id','=',False)]"
```

Comment:
I wouldn't do the change for everything, I would do it regarding it default security rules or the company_id field in the model. For instance, for `account.account` company_id is required, so the change won't make sense here IMO.

Reply:
Done :+1: 

I have kept the previous domain for all fields for which the related models have the company as a required field.

Reply:
Great, thanks! :+1: